RecoJets/JetProducers/plugins/MVAJetPuIdProducer.h

```diff
@@ -0,0 +1,49 @@
+#include "FWCore/Framework/interface/Event.h"
```

Comment:
please move the contents of this file to MVAJetPuIdProducer.cc
the MVAJetPuIdProducer.h by itself is not necessary.